bin/kafka-avro-console-consumer

```diff
@@ -16,7 +16,13 @@
 
 base_dir=$(dirname $0)/..
 
-CLASSPATH=$CLASSPATH:$base_dir/share/java/avro-serializer/*
+# Production jars
+export CLASSPATH=$CLASSPATH:$base_dir/share/java/kafka-serde-tools/*
+
+# Development jars. `mvn package` should collect all the required dependency jars here
```

Comment:
Because schema-registry shouldn't need to include both and I was treating `schema-registry-run-class` as intended for its namesake. I think continuing to just dump everything together is going down a dangerous path that'll eventually lead to conflicts. These scripts probably shouldn't even be using `schema-registry-run-class`... most of this is just fallout from having grouped the serializer into this repo even though it doesn't really fit well here, ended up causing annoying packaging issues, etc.

If we really want to go down that path and avoid this bit of duplication, we should use different `X-run-class` scripts for schema-registry and these serde-related scripts, even if both are just thin wrappers around an underlying generic `run-class` script.